diff --git a/ressources/python/pytson.py b/ressources/python/pytson.py
index 3eed890..06e0940 100644
--- a/ressources/python/pytson.py
+++ b/ressources/python/pytson.py
@@ -12,7 +12,7 @@ from PythonQt.QtGui import QApplication
 _PYTSON_VERSION = "1.2.3"
 
 
-def tr(context, sourcetext, *, disambiguation="", n=-1):
+def tr(context, sourcetext, disambiguation="", n=-1):
     """
     Returns the current translation for a string. This function calls can be
     extracted by pyTSon's pylupdate.py.
@@ -38,7 +38,7 @@ class Translatable(object):
     Baseclass for a class using translatable string literals.
     """
     @classmethod
-    def _tr(cls, sourcetext, *, disambiguation="", n=-1, context=None):
+    def _tr(cls, sourcetext, disambiguation="", n=-1, context=None):
         """
         Returns the current translation for a string. This method calls can be
         extracted by pyTSon's pylupdate.py.
diff --git a/ressources/python/pytsonui/__init__.py b/ressources/python/pytsonui/__init__.py
index 28be23d..f1a4c04 100644
--- a/ressources/python/pytsonui/__init__.py
+++ b/ressources/python/pytsonui/__init__.py
@@ -220,7 +220,7 @@ class UiLoader(QUiLoader):
     QUILoader subclass to omit the parent widget from being recreated.
     """
 
-    def __init__(self, main, parent=None, *, extraWidgets=None):
+    def __init__(self, main, parent=None, extraWidgets=None):
         """
         Instantiate a new object
         @param main: parent class which will be omitted
@@ -249,7 +249,7 @@ class UiLoader(QUiLoader):
             return QUiLoader.createWidget(self, clsname, parent, name)
 
 
-def setupUi(obj, uipath, *, widgets=None, seticons=True, iconpack=None,
+def setupUi(obj, uipath, widgets=None, seticons=True, iconpack=None,
             pluginicons=None, extraWidgets=None):
     """
     Loads a Qt designer file (.ui), creates the widgets defined in and adds
diff --git a/ressources/python/pytsonui/console.py b/ressources/python/pytsonui/console.py
index 580af9f..7402efa 100644
--- a/ressources/python/pytsonui/console.py
+++ b/ressources/python/pytsonui/console.py
@@ -37,7 +37,7 @@ class PythonConsoleDialog(QDialog):
     def __init__(self, tabcomplete=True, spaces=True, tabwidth=2,
                  font=defaultFont(), bgcolor=Qt.black, textcolor=Qt.white,
                  width=800, height=600, startup="", silentStartup=False,
-                 parent=None, *, catchstd=False):
+                 parent=None, catchstd=False):
         super(QDialog, self).__init__(parent)
         self.setAttribute(Qt.WA_DeleteOnClose)
 
@@ -55,7 +55,7 @@ class PythonConsole(QPlainTextEdit, pytson.Translatable):
     def __init__(self, tabcomplete=True, spaces=True, tabwidth=2,
                  font=defaultFont(), bgcolor=Qt.black, textcolor=Qt.white,
                  startup="", silentStartup=False,
-                 parent=None, *, catchstd=False, closeAction):
+                 parent=None, catchstd=False, closeAction):
         super(QPlainTextEdit, self).__init__(parent)
 
         self.setLineWrapMode(QPlainTextEdit.NoWrap)
diff --git a/ressources/python/ts3client.py b/ressources/python/ts3client.py
index 7584e07..c47619e 100644
--- a/ressources/python/ts3client.py
+++ b/ressources/python/ts3client.py
@@ -155,7 +155,7 @@ class IconPack(object):
         del cfg
         return IconPack(info=q.value(0))
 
-    def __init__(self, *, info=None, name=None):
+    def __init__(self, info=None, name=None):
         """
         Instantiates a new IconPack object referenced by its name or the
         internal info string (an Exception is raised if the iconpack could
diff --git a/ressources/python/ts3widgets/filetransfer.py b/ressources/python/ts3widgets/filetransfer.py
index 9addaaa..bc9ec3c 100644
--- a/ressources/python/ts3widgets/filetransfer.py
+++ b/ressources/python/ts3widgets/filetransfer.py
@@ -123,7 +123,7 @@ class FileListModel(QAbstractItemModel, pytson.Translatable):
     Itemmodel to abstract the files contained on a TS3 filepath.
     """
 
-    def __init__(self, schid, cid, password, parent=None, *, readonly=False):
+    def __init__(self, schid, cid, password, parent=None, readonly=False):
         super(QAbstractItemModel, self).__init__(parent)
 
         self.schid = schid
@@ -467,7 +467,7 @@ class FileBrowser(QDialog, pytson.Translatable):
     Dialog to display files contained on a TS3 filepath.
     """
 
-    def __init__(self, schid, cid, password='', path='/', parent=None, *,
+    def __init__(self, schid, cid, password='', path='/', parent=None,
                  staticpath=False, readonly=False, downloaddir=None,
                  iconpack=None):
         """
diff --git a/ressources/python/ts3widgets/serverview.py b/ressources/python/ts3widgets/serverview.py
index 829d6e5..6e5bc50 100644
--- a/ressources/python/ts3widgets/serverview.py
+++ b/ressources/python/ts3widgets/serverview.py
@@ -1391,7 +1391,7 @@ class Serverview(QTreeView):
     A QTreeView widget to display the complete view on a TS3 Server.
     """
 
-    def __init__(self, parent=None, *, schid=None):
+    def __init__(self, parent=None, schid=None):
         """
         Instantiates a new Serverview widget (including model and delegate).
         @param schid: the ID of the serverconnection
